### PR TITLE
Add initial implementation of protoc-gen-ts.

### PR DESCRIPTION
**Pull request by BenBirt**

### CODE REVIEW

File: protobufts/descriptor.proto [hunk 1]
```diff
@@ -0,0 +1,909 @@
+// Protocol Buffers - Google's data interchange format
```
Comment (lewish) [inline protobufts/descriptor.proto:1]: Are these modified at all? If not, can we not pull them in directly via bazel rather than adding to source?

Comment (lewish) [reply protobufts/descriptor.proto:1]: Like how we do this for e.g. `annotations.proto`

**Review by lewish — APPROVED** [review]
Approving for now but will take a longer pass when I can!

File: protobufts/descriptor.proto (hunk 1 above)
Comment (BenBirt) [reply protobufts/descriptor.proto:1]: nice suggestion, cleaned up a lot!

**Pull request merged**